packages/babel-parser/src/parser/expression.js

```diff
@@ -791,6 +791,13 @@ export default class ExpressionParser extends LValParser {
     return node;
   }
 
+  finishCoverCallExpressionArguments(
+    node: N.CallExpression | N.OptionalCallExpression,
+    isParenthesizedExpr?: boolean,
```

Comment:
Is `isParenthesizedExpr` ever used? If not we can simply remove it.

Reply:
It is in the flow plugin... I left it in just to keep all the sites consistent